crates/nil/src/handler.rs

```diff
@@ -222,7 +222,7 @@ pub(crate) fn formatting(
     }
 
     let Some(cmd) = &snap.config.formatting_command else {
-        return Ok(None);
+        bail!("No formatter configured. Set the nil.formatting.command LSP server setting.");
     };
```

Comment:
Now this can be:
```suggestion
let cmd = snap
    .config
    .formatting_command
    .as_ref()
    .context("No formatter configured. Set the nil.formatting.command LSP server setting.")?;
```

Reply:
Sounds good! It still prints the useful message with `context`:
```
[Error - 3:18:09 PM] Request textDocument/formatting failed.
  Message: No formatter configured. Set the nil.formatting.command LSP server setting.
  Code: -32603 
```